src/S3/MultipartUploadingTrait.php

```diff
@@ -102,8 +102,8 @@ protected function getInitiateParams()
             $params['ACL'] = $config['acl'];
         }
 
-        // Set the content type
-        if ($type = $this->getSourceMimeType()) {
+        // Set the content type if ContentType is not set
+        if (!isset($params['ContentType']) && $type = $this->getSourceMimeType()) {
```

Comment:
This `!isset(` should be `empty(`, otherwise we'll allow setting the content type to `''`.